GVFS/GVFS.Common/GVFSPlatform.cs

```diff
@@ -21,9 +21,11 @@ public GVFSPlatform(string executableExtension, string installerExtension)
         public abstract IGitInstallation GitInstallation { get; }
         public abstract IDiskLayoutUpgradeData DiskLayoutUpgrade { get; }
         public abstract IPlatformFileSystem FileSystem { get; }
-        public virtual bool IsUnderConstruction { get; } = false;
         public virtual bool SupportsGVFSService { get; } = true;
         public virtual bool SupportsGVFSUpgrade { get; } = true;
+        public virtual bool SupportsGVFSConfig { get; } = true;
+        public virtual bool SupportsKernelLogs { get; } = true;
+        public virtual bool UsesGitHooksLoader { get; } = false;

```

Comment:
Do you see these eventually being removed at some point when all the platforms we want to support have them?

Reply:
Yep all of these should be removed as the various platforms catch up in features.

I considered prefixing each of these properties with `UnderConstruction` to make that more clear. Thoughts?

Reply:
Take a look at my latest commit. I like that pattern because it makes it very obvious, both in the platform classes and at every usage site, that these are temporary flags. They are intended to be removed, and not intended to make features optional, for example.